<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PositionOrientation</class>
 <widget class="QWidget" name="PositionOrientation">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>40</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(0, 0, 0);</string>
  </property>
  <widget class="QLabel" name="RCS1000">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>10</y>
     <width>47</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>RCS2000</string>
   </property>
  </widget>
  <widget class="QLabel" name="Mark">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>10</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>l</string>
   </property>
  </widget>
  <widget class="QLabel" name="Heading">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>10</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>l</string>
   </property>
  </widget>
  <widget class="QLabel" name="Distance">
   <property name="geometry">
    <rect>
     <x>880</x>
     <y>10</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>l</string>
   </property>
  </widget>
  <widget class="QLabel" name="PitchAngle">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>10</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>l</string>
   </property>
  </widget>
  <widget class="QLabel" name="YawAngle">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>10</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>l</string>
   </property>
  </widget>
  <widget class="QLabel" name="RollAngle">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>10</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>l</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
 <slots>
  <slot>Exit()</slot>
 </slots>
</ui>
